cpython-unix/build-main.py

```diff
@@ -164,7 +165,7 @@ def main():
         release_tag = release_tag_from_git()
 
     # Guard against accidental misuse of the free-threaded flag with older versions
-    if "freethreaded" in args.options and python_majmin not in ("3.13",):
```

Comment:
We may want to change this to an open ended version compared to so we don't have to touch this every year.

Reply:
Won't it just become the only variant in Python 3.15?

I jest. I think you're right — I'll do that.